policyengine_us/parameters/gov/states/ri/tax/income/exemption/reduction/increment.yaml

```diff
@@ -0,0 +1,19 @@
+description: Rhode Island phases the exemption amount out at this increment.
+values:
+  2021-01-01: 6_000
+  2022-01-01: 6_200
+
+
+metadata:
+  label: Rhode Island exemption phase out increment
+  period: year
+  unit: currency-USD
+  reference:
+  # The tax form points to this worksheet: Exemption Worksheet for RI-1040NR, Page 1, line 6)
+  - title: Rhode Island Tax Rate Schedule and Worksheet 2022 (Exemption Worksheet line 6)
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20Tax%20Rate%20and%20Worksheets.pdf
+  # The tax form points to this worksheet: Exemption Worksheet for RI-1040 or RI-1040NR, Page 1, line 6)
+  - title: Rhode Island Tax Rate Schedule and Worksheet 2021 (Exemption Worksheet line 6)
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2021-11/2021-tax-rate-and-worksheets.pdf
+  - title: Rhode Island Legal Code R.I. Gen. Laws § 44-30-2.6 (E), (4), (a)
```

Comment:
```suggestion
  # Legal code specifies $2,500, inflation-adjusted starting in 1991.
  - title: Rhode Island Legal Code R.I. Gen. Laws § 44-30-2.6 (E), (4), (b)
```

Reply:
Legal code halves it for married filing separately, is there a different tax form for married filing separately?

Reply:
Neither the tax form or the worksheets specify the difference among filing status.

Reply:
Hmm do the forms otherwise specify instructions for married filing separately filers? If not it might suggest there are other materials for them. If so, we can explain that the revenue authority disregarded the law when creating the forms, and we can go with the forms instead.

Reply:
The worksheets specify "for all filing status types", and the tax forms do not have any guidance for separate filers.